Contract functionality adjustments (#23)
marinade-finance/validator-bonds: README.md

* [`programs/validator-bonds`](./programs/validator-bonds/) - Anchor on-chain contract project 
* [`packages/`](./packages/) - TypeScript packages related to on-chain program
  ([SDK](./packages/validator-bonds-sdk/), [CLI](./packages/validator-bonds-cli/))
* [`api/`](./api/) - in Rust developed OpenAPI service that publishes bonds data ([API endpoint](https://validator-bonds-api.marinade.finance/docs))

> * [`.buildkite/`](./.buildkite/) - automated pipelines that prepares data for bonds claiming, updating API data and similar

Reviewer: ```suggestion
* [`.buildkite/`](./.buildkite/) - automated pipelines that prepare data for bonds claiming, updating API data and similar
```

> * [`insurance-engine/`](./insurance-engine/) - under command of creating protected event data that is published in form of JSON and packed as a settlement on-chain

Reviewer: ```suggestion
* [`insurance-engine/`](./insurance-engine/) - code for a CLI creating protected event data that is published in form of JSON and packed as a settlement on-chain
```

* [`merkle-tree/`](./merkle-tree/) - generic Rust library implementing the merkle tree data structure management
* [`migrations/`](./migrations/) - SQL scripts to prepare and change DB schemas

> * [`scripts/`](./scripts/) - as the name suggests, scripts used in pipeline and to manage and integrate various repository parts

Reviewer: ```suggestion
* [`scripts/`](./scripts/) - scripts used in pipeline and to manage and integrate various repository parts
```

> * [`snapshot-parser`](./snapshot-parser/) - a service in command to parse Solana snapshots and provides data processed by `insurance-engine`

Reviewer: ```suggestion
* [`snapshot-parser`](./snapshot-parser/) - a CLI for parsing Solana snapshots and providing data to be processed by `insurance-engine`
```
